src/main/java/com/MyApp/budgetControl/domain/expense/InMemoryExpenseRepository.java

```diff
@@ -1,29 +1,35 @@
-package com.MyApp.budgetControl.domain.expense;
-
-import java.time.Instant;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.List;
-import java.util.UUID;
-
-abstract class InMemoryExpenseRepository implements ExpenseRepository {
-  Instant date = new Date().toInstant();
-
-  private final List<ExpenseEntity> expenses = new ArrayList<>(Arrays.asList(
-     new ExpenseEntity(UUID.randomUUID().toString(), 49.99, "groceries", "Biedronka market", date),
-     new ExpenseEntity(UUID.randomUUID().toString(), 10.00, "eating out ", "chinese food", date),
-     new ExpenseEntity(UUID.randomUUID().toString(), 150.50, "transport ", "Orlen", date)
-  ));
-
-  @Override
-  public final List<ExpenseEntity> findAll() {
-    return  expenses;
-  }
-
-  @Override
-  public ExpenseEntity save(ExpenseEntity newExpense) {
-    expenses.add(newExpense);
-    return  newExpense;
-  }
-}
+//package com.MyApp.budgetControl.domain.expense;
+//
+//import com.MyApp.budgetControl.domain.category.CategoryEntity;
+//import com.MyApp.budgetControl.domain.user.UserEntity;
+//
+//import java.time.Instant;
+//import java.util.ArrayList;
+//import java.util.Arrays;
+//import java.util.Collections;
+//import java.util.Date;
+//import java.util.HashSet;
+//import java.util.List;
+//import java.util.UUID;
+//
+//abstract class InMemoryExpenseRepository implements ExpenseRepository {
```

Comment:
Why its commented? If You dont need it then remove it, If You are going to change it later add some TODOs with comments

Reply:
Yes, it will not be used